docs/install.rst

```diff
@@ -18,6 +18,12 @@ Rally does not support Windows and is only actively tested on macOS and Linux. I
 Python
 ~~~~~~
 
+    .. note::
+
+        You don't need Python if you
```

Comment:
Is this line still needed?

Reply:
Oops I need to finish it to say that you don't need to install Python with the standalone method.